### PR TITLE
[DOCS] 8.0.0-alpha1 release notes

### PR DESCRIPTION
**Pull request by jakelandis**
Release notes for 8.0.0-alpha1

### Preview

https://elasticsearch_75761.docs-preview.app.elstc.co/guide/en/elasticsearch/reference/master/release-notes-8.0.0-alpha1.html

### CODE REVIEW

File: docs/reference/release-notes/8.0.0-alpha1.asciidoc [hunk 1]
```diff
@@ -1,51 +1,517 @@
...
+* [Rest Api Compatibility] Typed endpoints for search and related endpoints {es-pull}72155[#72155] (issues: {es-issue}51816[#51816], {es-issue}54160[#54160])
+
+Machine Learning::
+* Limit the input sequence length for NLP processors {es-pull}74905[#74905]
+* Merge the pytorch-inference feature branch {es-pull}73660[#73660] (issues: {es-issue}70713[#70713], {es-issue}71035[#71035], {es-issue}71177[#71177], {es-issue}71323[#71323], {es-issue}71679[#71679], {es-issue}72218[#72218], {es-issue}73523[#73523])
+* Parse time from inference result {es-pull}75570[#75570] (issue: {es-issue}75268[#75268])
+* Use configurable class labels for SA and NER results {es-pull}74473[#74473] (issues: {es-issue}74361[#74361], {es-issue}74401[#74401])
+* `PyTorch` Sequence Classification (Sentiment Analysis) task {es-pull}73764[#73764]
+
+SQL::
```
Comment (jakelandis) [inline docs/reference/release-notes/8.0.0-alpha1.asciidoc:512]: @elastic/es-ql - could someone help me properly label these as a `>bug`, `>enhancement`, or `>non-issue` (or other).

File: docs/reference/release-notes/8.0.0-alpha1.asciidoc [hunk 2]
```diff
@@ -1,51 +1,517 @@
...
+=== NOT CLASSIFIED
+
+Infra/Scripting::
+* Script: User funcs are script instance methods {es-pull}74268[#74268] (issues: {es-issue}68235[#68235], {es-issue}69742[#69742])
+
+MULTIPLE AREA LABELS::
+* [Rest Api Compatibility] Indicies boost in object format {es-pull}74422[#74422] (issues: {es-issue}51816[#51816], {es-issue}55078[#55078])
+* [Rest Api Compatibility] Typed endpoints for search and related endpoints {es-pull}72155[#72155] (issues: {es-issue}51816[#51816], {es-issue}54160[#54160])
+
+Machine Learning::
```
Comment (jakelandis) [inline docs/reference/release-notes/8.0.0-alpha1.asciidoc:505]: @elastic/machine-learning   - could someone help me properly label these as a `>bug`, `>enhancement`, or `>non-issue` (or other).

File: docs/reference/release-notes/8.0.0-alpha1.asciidoc [hunk 3]
```diff
@@ -1,51 +1,517 @@
...
+* Upgrade to Lucene 8.9.0 {es-pull}74729[#74729]
+
+
+
+[[other-8.0.0-alpha1]]
+[float]
+=== NOT CLASSIFIED
+
+Infra/Scripting::
+* Script: User funcs are script instance methods {es-pull}74268[#74268] (issues: {es-issue}68235[#68235], {es-issue}69742[#69742])
```
Comment (jakelandis) [inline docs/reference/release-notes/8.0.0-alpha1.asciidoc:499]: @stu-elastic - could you help me properly label these as a `>bug`, `>enhancement`, or `>non-issue` (or other).

File: docs/reference/release-notes/8.0.0-alpha1.asciidoc (hunk 2 above)
Comment (lcawl) [reply docs/reference/release-notes/8.0.0-alpha1.asciidoc]: I've made that change in faa3c866513f280b9d2a4a55be8f3360656b949b

File: docs/reference/release-notes/8.0.0-alpha1.asciidoc [hunk 4]
```diff
@@ -1,44 +1,520 @@
...
+* QL: Polish optimizer expression rule declaration {es-pull}71396[#71396]
+* QL: Propagate nullability constraints across conjunctions {es-pull}71187[#71187] (issue: {es-issue}70683[#70683])
+* [Rest Api Compatibility] Allow to use size -1 {es-pull}75342[#75342] (issues: {es-issue}51816[#51816], {es-issue}69548[#69548], {es-issue}70209[#70209])
+* [Rest Api Compatibility] Ignore `use_field_mapping` option for docvalue {es-pull}74435[#74435] (issue: {es-issue}55622[#55622])
+* [Rest Api Compatibility] `_time` and `_term` sort orders {es-pull}74919[#74919] (issues: {es-issue}39450[#39450], {es-issue}51816[#51816])
+
+Machine Learning::
+* Change version in serialization code in `TimingStats.java` to 7.4.0 {es-pull}44905[#44905] (issue: {es-issue}29857[#29857])
+* Limit the input sequence length for NLP processors {es-pull}74905[#74905]
+* Merge the pytorch-inference feature branch {es-pull}73660[#73660] (issues: {es-issue}70713[#70713], {es-issue}71035[#71035], {es-issue}71177[#71177], {es-issue}71323[#71323], {es-issue}71679[#71679], {es-issue}72218[#72218], {es-issue}73523[#73523])
```
Comment (droberts195) [inline docs/reference/release-notes/8.0.0-alpha1.asciidoc:342]:
```suggestion
```

I suggest removing this low level detail.  The functionality is better described by "Add a new application for evaluating PyTorch models..." a few lines below.  All this is part of the same change.

File: docs/reference/release-notes/8.0.0-alpha1.asciidoc [hunk 5]
```diff
@@ -1,44 +1,520 @@
...
+* Several minor Painless fixes {es-pull}61594[#61594]
+
+MULTIPLE AREA LABELS::
+* Avoid `StackOverflowError` due to regex alternate paths {es-pull}61259[#61259] (issue: {es-issue}60889[#60889])
+* Fix cat recovery display of bytes fields {es-pull}40379[#40379] (issue: {es-issue}40335[#40335])
+* Fixes to task result index mapping {es-pull}50359[#50359] (issue: {es-issue}50248[#50248])
+
+Machine Learning::
+* Checking if p-tasks metadata is null before updating state (#41091) {es-pull}41124[#41124] (issues: {es-issue}41090[#41090], {es-issue}41091[#41091])
+* Fix typo Korean HANGUL_JAMO {es-pull}74462[#74462]
```
Comment (davidkyle) [inline docs/reference/release-notes/8.0.0-alpha1.asciidoc:465]: I've labelled this one non-issue it is fixing a typo in a comment

File: docs/reference/release-notes/8.0.0-alpha1.asciidoc [hunk 6]
```diff
@@ -1,44 +1,520 @@
...
+* Fixes casting in constant folding {es-pull}61508[#61508]
+* Several minor Painless fixes {es-pull}61594[#61594]
+
+MULTIPLE AREA LABELS::
+* Avoid `StackOverflowError` due to regex alternate paths {es-pull}61259[#61259] (issue: {es-issue}60889[#60889])
+* Fix cat recovery display of bytes fields {es-pull}40379[#40379] (issue: {es-issue}40335[#40335])
+* Fixes to task result index mapping {es-pull}50359[#50359] (issue: {es-issue}50248[#50248])
+
+Machine Learning::
+* Checking if p-tasks metadata is null before updating state (#41091) {es-pull}41124[#41124] (issues: {es-issue}41090[#41090], {es-issue}41091[#41091])
```
Comment (davidkyle) [inline docs/reference/release-notes/8.0.0-alpha1.asciidoc:464]: This was a forward port from 6.7.2 but missing the v6.7.2 label which I've now added

File: docs/reference/release-notes/8.0.0-alpha1.asciidoc [hunk 7]
```diff
@@ -1,44 +1,520 @@
...
+* Fix compile error in `DataFrameTransformProgressIT` {es-pull}43099[#43099]
+
+
+
+[[upgrade-8.0.0-alpha1]]
+[float]
+=== Upgrades
+
+NOT CLASSIFIED::
+* Upgrade to Lucene 8.8.2 on 8.0 {es-pull}71587[#71587]
```
Comment (droberts195) [inline docs/reference/release-notes/8.0.0-alpha1.asciidoc:499]:
```suggestion
```

This seems to be superseded by the entry below, and it's just confusing to release note intermediate steps that were never released.

File: docs/reference/release-notes/8.0.0-alpha1.asciidoc [hunk 8]
```diff
@@ -1,44 +1,520 @@
...
+* Fix `VectorsFeatureSetUsage` serialization in BWC mode {es-pull}55399[#55399] (issue: {es-issue}55378[#55378])
+* Handle total hits equal to `track_total_hits` {es-pull}37907[#37907] (issue: {es-issue}37897[#37897])
+* Improve error msg for CCS request on node without remote cluster role {es-pull}60351[#60351] (issue: {es-issue}59683[#59683])
+
+Snapshot/Restore::
+* Fix GET /_snapshot/_all/_all if there are no repos {es-pull}43558[#43558] (issue: {es-issue}43547[#43547])
+
+Transform::
+* Fix compile error in `DataFrameTransformProgressIT` {es-pull}43099[#43099]
+
```
Comment (droberts195) [inline docs/reference/release-notes/8.0.0-alpha1.asciidoc:491]:
```suggestion
```

Internal detail that end users don't care about.

File: docs/reference/release-notes/8.0.0-alpha1.asciidoc [hunk 9]
```diff
@@ -1,44 +1,520 @@
...
+MULTIPLE AREA LABELS::
+* Avoid `StackOverflowError` due to regex alternate paths {es-pull}61259[#61259] (issue: {es-issue}60889[#60889])
+* Fix cat recovery display of bytes fields {es-pull}40379[#40379] (issue: {es-issue}40335[#40335])
+* Fixes to task result index mapping {es-pull}50359[#50359] (issue: {es-issue}50248[#50248])
+
+Machine Learning::
+* Checking if p-tasks metadata is null before updating state (#41091) {es-pull}41124[#41124] (issues: {es-issue}41090[#41090], {es-issue}41091[#41091])
+* Fix typo Korean HANGUL_JAMO {es-pull}74462[#74462]
+* Handle null value of `FieldCapabilitiesResponse` {es-pull}64327[#64327]
+* Relax the test assertion so that the test always passes {es-pull}44133[#44133] (issue: {es-issue}44132[#44132])
```
Comment (droberts195) [inline docs/reference/release-notes/8.0.0-alpha1.asciidoc:467]:
```suggestion
```

PR should have been labeled `>test`, not `>bug`.

File: docs/reference/release-notes/8.0.0-alpha1.asciidoc (hunk 6 above)
Comment (droberts195) [inline docs/reference/release-notes/8.0.0-alpha1.asciidoc:464]:
```suggestion
* Checking if persistent tasks metadata is null before updating state (#41091) {es-pull}41124[#41124] (issues: {es-issue}41090[#41090], {es-issue}41091[#41091])
```

File: docs/reference/release-notes/8.0.0-alpha1.asciidoc [hunk 10]
```diff
@@ -1,44 +1,520 @@
...
+* Remove erroneous licence {es-pull}58732[#58732] (issue: {es-issue}58682[#58682])
```
Comment (droberts195) [inline docs/reference/release-notes/8.0.0-alpha1.asciidoc:468]:
```suggestion
```

Internal mistake that was never released - should have been labelled `>non-issue`.

File: docs/reference/release-notes/8.0.0-alpha1.asciidoc [hunk 11]
```diff
@@ -1,44 +1,520 @@
...
+* Merge the pytorch-inference feature branch {es-pull}73660[#73660] (issues: {es-issue}70713[#70713], {es-issue}71035[#71035], {es-issue}71177[#71177], {es-issue}71323[#71323], {es-issue}71679[#71679], {es-issue}72218[#72218], {es-issue}73523[#73523])
+* Parse time from inference result {es-pull}75570[#75570] (issue: {es-issue}75268[#75268])
+* Use configurable class labels for SA and NER results {es-pull}74473[#74473] (issues: {es-issue}74361[#74361], {es-issue}74401[#74401])
+* `PyTorch` Sequence Classification (Sentiment Analysis) task {es-pull}73764[#73764]
 * The Windows build platform for the {ml} C++ code now uses Visual Studio 2019 {ml-pull}1352[#1352]
 * The macOS build platform for the {ml} C++ code is now Mojave running Xcode 11.3.1,
   or Ubuntu 20.04 running clang 8 for cross compilation {ml-pull}1429[#1429]
 * The Linux build platform for the {ml} C++ code is now CentOS 7 running gcc 9.3 {ml-pull}1170[#1170]
 * Add a new application for evaluating PyTorch models. The app depends on LibTorch - the C++ front end to PyTorch and performs inference on models stored in the TorchScript format {ml-pull}1902[#1902]
```
Comment (droberts195) [inline docs/reference/release-notes/8.0.0-alpha1.asciidoc:350]:
```suggestion
* Add a new application for evaluating PyTorch models. The app depends on LibTorch - the C++ front end to PyTorch - and performs inference on models stored in the TorchScript format {ml-pull}1902[#1902]
```

File: docs/reference/release-notes/8.0.0-alpha1.asciidoc [hunk 12]
```diff
@@ -1,51 +1,517 @@
...
+
+Machine Learning::
+* Limit the input sequence length for NLP processors {es-pull}74905[#74905]
+* Merge the pytorch-inference feature branch {es-pull}73660[#73660] (issues: {es-issue}70713[#70713], {es-issue}71035[#71035], {es-issue}71177[#71177], {es-issue}71323[#71323], {es-issue}71679[#71679], {es-issue}72218[#72218], {es-issue}73523[#73523])
+* Parse time from inference result {es-pull}75570[#75570] (issue: {es-issue}75268[#75268])
+* Use configurable class labels for SA and NER results {es-pull}74473[#74473] (issues: {es-issue}74361[#74361], {es-issue}74401[#74401])
+* `PyTorch` Sequence Classification (Sentiment Analysis) task {es-pull}73764[#73764]
+
+SQL::
+* Reference relevant issue in `ReplaceSumWithStats` {es-pull}74396[#74396] (issues: {es-issue}45251[#45251], {es-issue}71582[#71582])
```
Comment (Luegg) [inline docs/reference/release-notes/8.0.0-alpha1.asciidoc:516]:
```suggestion
```

PR should have been labeled ">non-issue"

File: docs/reference/release-notes/8.0.0-alpha1.asciidoc [hunk 13]
```diff
@@ -1,44 +1,520 @@
...
+Infra/Scripting::
+* Script: User funcs are script instance methods {es-pull}74268[#74268] (issues: {es-issue}68235[#68235], {es-issue}69742[#69742])
+
+MULTIPLE AREA LABELS::
+* [Rest Api Compatibility] Indicies boost in object format {es-pull}74422[#74422] (issues: {es-issue}51816[#51816], {es-issue}55078[#55078])
+* [Rest Api Compatibility] Typed endpoints for search and related endpoints {es-pull}72155[#72155] (issues: {es-issue}51816[#51816], {es-issue}54160[#54160])
+
+SQL::
+* Reference relevant issue in `ReplaceSumWithStats` {es-pull}74396[#74396] (issues: {es-issue}45251[#45251], {es-issue}71582[#71582])
+* Update supported version for `date_nanos` {es-pull}68214[#68214] (issues: {es-issue}67666[#67666], {es-issue}68198[#68198])
```
Comment (Luegg) [inline docs/reference/release-notes/8.0.0-alpha1.asciidoc:517]:
```suggestion
* Support for `date_nanos` data type {es-pull}67666[#67666], {es-pull}68214[#68214] (issues: {es-issue}38562[#38562])
```

I think #67666 is the relevant PR and it is labeled with `>feature`.

File: docs/reference/release-notes/8.0.0-alpha1.asciidoc [hunk 14]
```diff
@@ -1,44 +1,520 @@
+:es-issue: https://github.com/elastic/elasticsearch/issues/
+:es-pull:  https://github.com/elastic/elasticsearch/pull/
+
```
Comment (jrodewig) [inline docs/reference/release-notes/8.0.0-alpha1.asciidoc:3]: We pull this in from an attributes file in the `elastic/docs` repo. We don't need it here.
```suggestion
```

File: docs/reference/release-notes/8.0.0-alpha1.asciidoc [hunk 15]
```diff
@@ -1,44 +1,520 @@
...
+Infra/Scripting::
+* Consolidate script parsing from object {es-pull}59507[#59507] (issue: {es-issue}59391[#59391])
+* Scripting: Move `script_cache` into _nodes/stats {es-pull}59265[#59265] (issues: {es-issue}50152[#50152], {es-issue}59262[#59262])
+* Scripting: Remove general cache settings {es-pull}59262[#59262] (issue: {es-issue}50152[#50152])
+
+Infra/Settings::
+* Change default value of `action.destructive_requires_name` to True {es-pull}66908[#66908] (issue: {es-issue}61074[#61074])
+* Forbid settings without a namespace {es-pull}45947[#45947] (issues: {es-issue}45905[#45905], {es-issue}45940[#45940])
+
+MULTIPLE AREA LABELS::
```
Comment (jrodewig) [inline docs/reference/release-notes/8.0.0-alpha1.asciidoc:117]: We should assign the PRs in this section to one of the areas.

File: docs/reference/release-notes/8.0.0-alpha1.asciidoc [hunk 16]
```diff
@@ -1,44 +1,520 @@
...
+
+Infra/Core::
+* Remove XPackClient from x-pack {es-pull}42729[#42729]
+* Remove the transport client {es-pull}42538[#42538]
+* Remove transport client from xpack {es-pull}42202[#42202]
+
+Infra/REST API::
+* Copy http headers to `ThreadContext` strictly {es-pull}45945[#45945]
+
+MULTIPLE AREA LABELS::
```
Comment (jrodewig) [inline docs/reference/release-notes/8.0.0-alpha1.asciidoc:227]: We should assign the PRs in this section to one of the areas.

File: docs/reference/release-notes/8.0.0-alpha1.asciidoc [hunk 17]
```diff
@@ -1,44 +1,520 @@
...
+
+Infra/Settings::
+* Fixed inconsistent Setting.exist() {es-pull}46603[#46603] (issue: {es-issue}41830[#41830])
+* Remove setting `index.optimize_auto_generated_id` (#27583) {es-pull}27600[#27600] (issue: {es-issue}27583[#27583])
+
+License::
+* Add deprecated `accept_enterprise` param to /_xpack {es-pull}58220[#58220] (issue: {es-issue}58217[#58217])
+* Support "accept_enterprise" param in get license {es-pull}50067[#50067] (issue: {es-issue}49474[#49474])
+
+MULTIPLE AREA LABELS::
```
Comment (jrodewig) [inline docs/reference/release-notes/8.0.0-alpha1.asciidoc:328]: We should assign the PRs in this section to one of the areas.

File: docs/reference/release-notes/8.0.0-alpha1.asciidoc [hunk 18]
```diff
@@ -1,44 +1,520 @@
...
+* Handle incorrect header values {es-pull}64708[#64708] (issues: {es-issue}51816[#51816], {es-issue}64689[#64689])
+* Ignore media ranges when parsing {es-pull}64721[#64721] (issues: {es-issue}51816[#51816], {es-issue}64689[#64689])
+* `RestController` should not consume request content {es-pull}44902[#44902] (issue: {es-issue}37504[#37504])
+
+Infra/Scripting::
+* Change compound assignment structure to support String concatenation {es-pull}61825[#61825]
+* Fixes casting in constant folding {es-pull}61508[#61508]
+* Several minor Painless fixes {es-pull}61594[#61594]
+
+MULTIPLE AREA LABELS::
```
Comment (jrodewig) [inline docs/reference/release-notes/8.0.0-alpha1.asciidoc:458]: We should assign the PRs in this section to one of the areas.

File: docs/reference/release-notes/8.0.0-alpha1.asciidoc [hunk 19]
```diff
@@ -1,44 +1,520 @@
...
+
+
+[[other-8.0.0-alpha1]]
+[float]
+=== NOT CLASSIFIED
+
+Infra/Scripting::
+* Script: User funcs are script instance methods {es-pull}74268[#74268] (issues: {es-issue}68235[#68235], {es-issue}69742[#69742])
+
+MULTIPLE AREA LABELS::
```
Comment (jrodewig) [inline docs/reference/release-notes/8.0.0-alpha1.asciidoc:511]: We should assign the PRs in this section to one of the areas.

**Review by jrodewig — COMMENTED** [review]
Thanks for putting this together @jakelandis. I left some feedback, but most of them are nits.

I'll take another look once we get all the multi-assigned and unassigned PRs in place. Thanks again!

File: docs/reference/release-notes/8.0.0-alpha1.asciidoc [hunk 20]
```diff
@@ -1,44 +1,520 @@
...
+Cluster Coordination::
+* Remove join timeout {es-pull}60873[#60873] (issue: {es-issue}60872[#60872])
+* Remove node filters for voting config exclusions {es-pull}55673[#55673] (issues: {es-issue}47990[#47990], {es-issue}50836[#50836])
+* Remove support for delaying state recovery pending master {es-pull}53845[#53845] (issue: {es-issue}51806[#51806])
+
+Distributed::
+* Goodbye and thank you synced flush! {es-pull}50882[#50882] (issues: {es-issue}50776[#50776], {es-issue}50835[#50835])
+
+EQL::
+* [DOCS] EQL: Remove `wildcard` function {es-pull}72121[#72121] (issues: {es-issue}71906[#71906], {es-issue}72119[#72119])
```
Comment (jrodewig) [inline docs/reference/release-notes/8.0.0-alpha1.asciidoc:46]: This is a docs PR. I've fixed the labelling.
```suggestion
```

File: docs/reference/release-notes/8.0.0-alpha1.asciidoc [hunk 21]
```diff
@@ -1,44 +1,520 @@
...
+
 
 [[enhancement-8.0.0-alpha1]]
 [float]
 === Enhancements
 
-Machine learning::
+Analysis::
+* Moving `reload_analyzers` endpoint to xpack {es-pull}43559[#43559]
+* [Docs] Move mentions of updateable synonyms flag {es-pull}43536[#43536]
```
Comment (jrodewig) [inline docs/reference/release-notes/8.0.0-alpha1.asciidoc:255]: This is a docs PR.
```suggestion
```

File: docs/reference/release-notes/8.0.0-alpha1.asciidoc [hunk 22]
```diff
@@ -1,44 +1,520 @@
...
+[[other-8.0.0-alpha1]]
+[float]
+=== NOT CLASSIFIED
+
+Infra/Scripting::
+* Script: User funcs are script instance methods {es-pull}74268[#74268] (issues: {es-issue}68235[#68235], {es-issue}69742[#69742])
+
+MULTIPLE AREA LABELS::
+* [Rest Api Compatibility] Indicies boost in object format {es-pull}74422[#74422] (issues: {es-issue}51816[#51816], {es-issue}55078[#55078])
+* [Rest Api Compatibility] Typed endpoints for search and related endpoints {es-pull}72155[#72155] (issues: {es-issue}51816[#51816], {es-issue}54160[#54160])
```
Comment (jrodewig) [inline docs/reference/release-notes/8.0.0-alpha1.asciidoc:513]: Nits
```suggestion
* [REST API Compatibility] Indicies boost in object format {es-pull}74422[#74422] (issues: {es-issue}51816[#51816], {es-issue}55078[#55078])
* [REST API Compatibility] Typed endpoints for search and related endpoints {es-pull}72155[#72155] (issues: {es-issue}51816[#51816], {es-issue}54160[#54160])
```

File: docs/reference/release-notes/8.0.0-alpha1.asciidoc [hunk 23]
```diff
@@ -1,44 +1,520 @@
...
+* [Rest Api Compatibility] Typed endpoints for explain api {es-pull}73901[#73901] (issue: {es-issue}51816[#51816])
+* [Rest Api Compatibility] Typed endpoints for search `_count` api {es-pull}73958[#73958] (issues: {es-issue}42112[#42112], {es-issue}51816[#51816])
+* [Rest Api Compatibility] Typed indexing stats {es-pull}74181[#74181] (issues: {es-issue}47203[#47203], {es-issue}51816[#51816], {es-issue}54160[#54160])
+* [Rest Api Compatibility] Types for Percolate Query Api {es-pull}74698[#74698] (issues: {es-issue}46985[#46985], {es-issue}51816[#51816], {es-issue}54160[#54160], {es-issue}74689[#74689])
+* [Rest Api Compatibility] Validate Query typed api {es-pull}74171[#74171] (issues: {es-issue}46927[#46927], {es-issue}51816[#51816], {es-issue}54160[#54160])
+* [Rest Api Compatibility] Voting config exclusion exception message {es-pull}75406[#75406] (issues: {es-issue}51816[#51816], {es-issue}55291[#55291])
+* [Rest Api Compatibility] `MoreLikeThisQuery` with types {es-pull}75123[#75123] (issues: {es-issue}42198[#42198], {es-issue}51816[#51816], {es-issue}54160[#54160])
+* [Rest Compatible Api] update and delete by query using size field {es-pull}69606[#69606]
```
Comment (jrodewig) [inline docs/reference/release-notes/8.0.0-alpha1.asciidoc:315]: Nits
```suggestion
* [REST API Compatibility] Typed endpoints for Index and Get APIs {es-pull}69131[#69131] (issue: {es-issue}54160[#54160])
* [REST API Compatibility] Typed endpoints for Put and Get Mapping and get field mappings {es-pull}71721[#71721] (issues: {es-issue}51816[#51816], {es-issue}54160[#54160])
* [REST API Compatibility] Allow `copy_settings` flag for resize operations {es-pull}75184[#75184] (issues: {es-issue}38514[#38514], {es-issue}51816[#51816])
* [REST API Compatibility] Allow for type in geo shape query {es-pull}74553[#74553] (issues: {es-issue}51816[#51816], {es-issue}54160[#54160])
* [REST API Compatibility] Always return `adjust_pure_negative` value {es-pull}75182[#75182] (issues: {es-issue}49543[#49543], {es-issue}51816[#51816])
* [REST API Compatibility] Clean up x-pack/plugin rest compat tests {es-pull}74701[#74701] (issue: {es-issue}51816[#51816])
* [REST API Compatibility] Do not return `_doc` for empty mappings in template {es-pull}75448[#75448] (issues: {es-issue}51816[#51816], {es-issue}54160[#54160], {es-issue}70966[#70966], {es-issue}74544[#74544])
* [REST API Compatibility] Dummy REST action for `indices.upgrade` api {es-pull}75136[#75136] (issue: {es-issue}51816[#51816])
* [REST API Compatibility] REST Terms vector typed response {es-pull}73117[#73117]
* [REST API Compatibility] Rename `BulkItemResponse.Failure` type field {es-pull}74937[#74937] (issue: {es-issue}51816[#51816])
* [REST API Compatibility] Type metadata for docs used in simulate request {es-pull}74222[#74222] (issues: {es-issue}51816[#51816], {es-issue}54160[#54160])
* [REST API Compatibility] Typed `TermLookups` {es-pull}74544[#74544] (issues: {es-issue}46943[#46943], {es-issue}51816[#51816], {es-issue}54160[#54160])
* [REST API Compatibility] Typed and xpack graph explore api {es-pull}74185[#74185] (issues: {es-issue}46935[#46935], {es-issue}51816[#51816], {es-issue}54160[#54160])
* [REST API Compatibility] Typed endpoint for bulk api {es-pull}73571[#73571] (issue: {es-issue}51816[#51816])
* [REST API Compatibility] Typed endpoint for multiget api {es-pull}73878[#73878] (issue: {es-issue}51816[#51816])
* [REST API Compatibility] Typed endpoints for `RestUpdateAction` and `RestDeleteAction` {es-pull}73115[#73115] (issues: {es-issue}51816[#51816], {es-issue}54160[#54160])
* [REST API Compatibility] Typed endpoints for `get_source` api {es-pull}73957[#73957] (issues: {es-issue}46587[#46587], {es-issue}46931[#46931], {es-issue}51816[#51816])
* [REST API Compatibility] Typed endpoints for explain api {es-pull}73901[#73901] (issue: {es-issue}51816[#51816])
* [REST API Compatibility] Typed endpoints for search `_count` api {es-pull}73958[#73958] (issues: {es-issue}42112[#42112], {es-issue}51816[#51816])
* [REST API Compatibility] Typed indexing stats {es-pull}74181[#74181] (issues: {es-issue}47203[#47203], {es-issue}51816[#51816], {es-issue}54160[#54160])
* [REST API Compatibility] Types for Percolate Query Api {es-pull}74698[#74698] (issues: {es-issue}46985[#46985], {es-issue}51816[#51816], {es-issue}54160[#54160], {es-issue}74689[#74689])
* [REST API Compatibility] Validate Query typed api {es-pull}74171[#74171] (issues: {es-issue}46927[#46927], {es-issue}51816[#51816], {es-issue}54160[#54160])
* [REST API Compatibility] Voting config exclusion exception message {es-pull}75406[#75406] (issues: {es-issue}51816[#51816], {es-issue}55291[#55291])
* [REST API Compatibility] `MoreLikeThisQuery` with types {es-pull}75123[#75123] (issues: {es-issue}42198[#42198], {es-issue}51816[#51816], {es-issue}54160[#54160])
* [REST API Compatibility] Update and delete by query using size field {es-pull}69606[#69606]
```

File: docs/reference/release-notes/8.0.0-alpha1.asciidoc [hunk 24]
```diff
@@ -1,44 +1,520 @@
...
+MULTIPLE AREA LABELS::
+* Add recovery state tracking for Searchable Snapshots {es-pull}60505[#60505]
+* Configure security for the initial node cli {es-pull}74868[#74868]
+* QL: Improve removal of items during iteration {es-pull}71193[#71193]
+* QL: Make canonical form take into account children {es-pull}71266[#71266]
+* QL: Polish optimizer expression rule declaration {es-pull}71396[#71396]
+* QL: Propagate nullability constraints across conjunctions {es-pull}71187[#71187] (issue: {es-issue}70683[#70683])
+* [Rest Api Compatibility] Allow to use size -1 {es-pull}75342[#75342] (issues: {es-issue}51816[#51816], {es-issue}69548[#69548], {es-issue}70209[#70209])
+* [Rest Api Compatibility] Ignore `use_field_mapping` option for docvalue {es-pull}74435[#74435] (issue: {es-issue}55622[#55622])
+* [Rest Api Compatibility] `_time` and `_term` sort orders {es-pull}74919[#74919] (issues: {es-issue}39450[#39450], {es-issue}51816[#51816])
```
Comment (jrodewig) [inline docs/reference/release-notes/8.0.0-alpha1.asciidoc:337]:
```suggestion
* [REST API Compatibility] Allow to use size -1 {es-pull}75342[#75342] (issues: {es-issue}51816[#51816], {es-issue}69548[#69548], {es-issue}70209[#70209])
* [REST API Compatibility] Ignore `use_field_mapping` option for docvalue {es-pull}74435[#74435] (issue: {es-issue}55622[#55622])
* [REST API Compatibility] `_time` and `_term` sort orders {es-pull}74919[#74919] (issues: {es-issue}39450[#39450], {es-issue}51816[#51816])
```

File: docs/reference/release-notes/8.0.0-alpha1.asciidoc [hunk 25]
```diff
@@ -1,44 +1,520 @@
...
 * The macOS build platform for the {ml} C++ code is now Mojave running Xcode 11.3.1,
   or Ubuntu 20.04 running clang 8 for cross compilation {ml-pull}1429[#1429]
 * The Linux build platform for the {ml} C++ code is now CentOS 7 running gcc 9.3 {ml-pull}1170[#1170]
 * Add a new application for evaluating PyTorch models. The app depends on LibTorch - the C++ front end to PyTorch and performs inference on models stored in the TorchScript format {ml-pull}1902[#1902]
+
+Mapping::
+* Sparse vector to throw exception consistently {es-pull}62646[#62646]
+
+NOT CLASSIFIED::
+* [REST Compatible API] 'template' parameter and field on PUT index template {es-pull}71238[#71238] (issues: {es-issue}49460[#49460], {es-issue}51816[#51816], {es-issue}68905[#68905])
```
Comment (jrodewig) [inline docs/reference/release-notes/8.0.0-alpha1.asciidoc:356]:
```suggestion
* [REST API Compatability] 'template' parameter and field on PUT index template {es-pull}71238[#71238] (issues: {es-issue}49460[#49460], {es-issue}51816[#51816], {es-issue}68905[#68905])
```

File: docs/reference/release-notes/8.0.0-alpha1.asciidoc (hunk 3 above)
Comment (stu-elastic) [reply docs/reference/release-notes/8.0.0-alpha1.asciidoc:502]: they are internal work for a feature but not a feature themselves `>non-issue`?

File: docs/reference/release-notes/8.0.0-alpha1.asciidoc (hunk 5 above)
Comment (lcawl) [reply docs/reference/release-notes/8.0.0-alpha1.asciidoc]: Thanks! I removed it in 09b171fe13f5e853168f68d0d6d0f36f401dbcf0

File: docs/reference/release-notes/8.0.0-alpha1.asciidoc (hunk 6 above)
Comment (lcawl) [reply docs/reference/release-notes/8.0.0-alpha1.asciidoc]: Thanks!  I've removed this one in 09b171fe13f5e853168f68d0d6d0f36f401dbcf0 too

**Comment by jakelandis** [comment]
Thanks everyone for the help !  I think these are looking good now @jrodewig or @lcawl - would mind taking a final look (feel free to push the merge button).

**Review by jrodewig — APPROVED** [review]
LGTM. I'll merge this in. Thanks @jakelandis.

**Pull request merged**